community/modules/scripts/ramble-execute/variables.tf

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "ramble_path" {
+  description = "Path to the Ramble installation"
+  default     = null
+  type        = string
+}
+
+variable "log_file" {
+  description = "Log file to write output from Ramble execute steps into"
+  default     = "/var/log/ramble-execute.log"
+  type        = string
+}
+
+variable "spack_path" {
+  description = "Path to the Spack installation"
+  default     = null
```

Comment:
IIUC, spack path and ramble path are required. There should be no default, not even null, for required variables.